README.md

```diff
@@ -253,17 +253,44 @@ If you want to restrict navigation between your slides (i.e. the user has to tog
 All you have to do is implement `ISlidePolicy` in your slides:
 ```java
 public final class MySlide extends Fragment implements ISlidePolicy {
+/* Usage Example part 1 (Create a public variable for the CheckBox):
+ * private CheckBox checkBox;
+ */
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.myslide_layout, container, false);
+	
+	/* Usage Example part 2 (Assign the CheckBox view to the variable) :
+	 * checkBox = view.findViewById(R.id.checkboxid);
+	 */
```

Comment:
This whole part can probably go.

Reply:
I meant to make changes to the documentation understandable even by newbie developers. The whole pull request was meant to just give the user a full step by step guide.